tutorials/colors/colormapnorms.py

```diff
@@ -312,7 +313,8 @@ def _inverse(x):
 # ----------------------------------------------------------
 #
 # The `.TwoSlopeNorm` described above makes a useful example for
-# defining your own norm.
+# defining your own norm.  Note for the colorbar to work, you must
+# define an inverse for your norm:
```

Comment:
I wonder if it would make sense to create an abstract base class for Norms that raises `NotImplemented` for `inverse()` so that it doesn't accidentally work with a colorbar? (I think right now it is inheriting the inverse from the standard "Linear" `Normalize` class. We could catch the `NotImplemented` in the Colorbar code and warn appropriately there that the inverse method needs to be implemented.

Reply:
I think that would be sensible.

Reply:
#7320 seems related to that.